# <center>*Curriculum Vitae*</center>

## <center>Phillip <u>Armand</u> Bester</center>

| Email addresses                                              | Online presence                                              | HPCSA                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
| <img align="left" src="media/email.png" style="zoom:20%;" /> | <img align="centre" src="media/qrcode.png" style="zoom:31%;" /> | <img align="centre" src="media/HPCSA_qr.png" style="zoom:10%;" /> |

| Personal details |                       |
|:---------------- |:--------------------- |
| Surname          | Bester                |
| Names            | Phillip <u>Armand</u> |
| Identity no      | 8112235032088         |
| Gender           | Male                  |
| Nationality      | South African         |
| Criminal record  | None                  |
| Languages        | English and Afrikaans |

#### Health Professions Council of South Africa registration

Date Obtained: 01/12/2014

Category:            Virology

Number:             MS 0001511

### Education

- **2007 M.Sc**      Distinction UFS.  Department of Biotechnology  

- **2004 B.Sc.**      Honours  UFS. Biochemistry  

- **2003 B.Sc**       UFS.  Biochemistry

### Experience

#### Timeline

- **2004  - 2010**    Practical Assistant for Undergraduate practicals (UFS)    

- **2005**                 Part-time Lecturer (Central University of Technology)        

- **2007 - 2010**     Research Assistant for the National Earthquake Laboratory of South African Mines 

- **2011- 2015**     Assistant Researcher.  Development and application of a HIV resistance genotyping assay. Data analysis and management of collected genotypes and clinical information. 

- **2015 - current**  Medical scientist.  Virology, NHLS Universitas, Bloemfontein 

#### Teaching experience at NHLS/UFS

- Main and co- supervision for Honours and M. students 
  
  - Main supervisor for Honours student developing an HIV integrase drug resistance assay.
  
  - Equally shared supervisor for two Honours students on: A simple and rapid approach to prepare Sindbis and West Nile viral RNA controls for differentiation between positive samples and laboratory contamination as shown in the publication list.
  
  - Co-supervisor for M. Med student looking at the influence of volumes on dry blood spot samples for infant HIV diagnostics.
  
  - Co-supervision of M. Med student looking at HBV drug resistance and viral loads

- Marking of Honours and M. students dissertations
  
  - Marking of M. Sc thesis for Prof Gert van Zyl, Stellenbosch University

- Lecturing Honours students and medical registrars in bioinformatics (http://vdeb.hpc.ufs.ac.za/)
  
- University of the Free State Medical Faculty Scientific committee member
  

#### Academic background

M.Sc in biochemistry with experience in molecular biology, including nucleic acid extraction, polymerase chain reaction and designing of PCR primers, recombinant DNA techniques, cloning, site directed mutagenesis, DNA sequencing, recombinant protein expression and enzyme kinetics.

#### Ongoing and recent academic projects (2020-2021)

##### HBV

We are in collaboration with an Oxford group (Wellcome Trust funding under Oxford PI Prof Philippa Matthews) with a few Hepatitis B virus projects. One of our projects was a systematic review of HBV seroepidemiology in Africa.  Here I was involved in visualizations one of which is available at https://hbv-geo.shinyapps.io/oxafricahbv/.

We are analysing retrospective data from the NHLS Central Data Warehouse with the main focus on HBV infection stratified by other factors such as HIV status, gender, age and health outcomes.

##### HIV

Co-supervision M. Med with the aim of determining the effect of volume on dry blood spot cards used for infant HIV diagnosis. (M. Med awarded)

Co-supervision of HIV drug resistance mutations K65R and M184V to determine if these mutations can reside on the same virus by using a cloning strategy. This was part of an HPCSA portfolio for an intern.

##### SARS-COV2

Next generation sequencing and surveillance of SARS-COV2 genomes where I am involved in sample selection, bio-informatics and genome consensus generation and quality control (MRC funding as a member of the Network for Genomic Surveillance in South Africa).

Seroepidemiology from HIV viral load samples using repeated cross-sectional sampling of approximately 5000 samples.

### Academic

#### Peer reviewed publications

* Peter Mwangi, Milton Mogotsi1 Ayodeji Ogunbayo, Teboho Mooko, Wairimu Maringa, Hlengiwe Sondlane, Kelebogile Nkwadipo, Olusesan Adelabu, **Philip Armand Bester**, Dominique Goedhals, Martin Nyaga. A decontamination strategy for resolving SARS‑CoV‑2 amplicon contamination in a next‑generation sequencing laboratory.  2022. Archives of Virology. 10.1007/s00705-022-05411-z

* Raquel Viana, Sikhulile Moyo, Daniel G. Amoako, Houriiyah Tegally, Cathrine Scheepers, Christian L. Althaus, Ugochukwu J. Anyaneji, **Phillip A. Bester**, Maciej F. Boni, Mohammed Chand, Wonderful T. Choga, Rachel Colquhoun, Michaela Davids, Koen Deforche, Deelan Doolabh, Louis du Plessis, Susan Engelbrecht, Josie Everatt, Jennifer Giandhari, Marta Giovanetti, Diana Hardie, Verity Hill, Nei-Yuan Hsiao, Arash Iranzadeh, Arshad Ismail, Charity Joseph, Rageema Joseph, Legodile Koopile, Sergei L. Kosakovsky Pond, Moritz U. G. Kraemer, Lesego Kuate-Lere, Oluwakemi Laguda-Akingba, Onalethatha Lesetedi-Mafoko, Richard J. Lessells, Shahin Lockman, Alexander G. Lucaci, Arisha Maharaj, Boitshoko Mahlangu, Tongai Maponga, Kamela Mahlakwane, Zinhle Makatini, Gert Marais, Dorcas Maruapula, Kereng Masupu, Mogomotsi Matshaba, Simnikiwe Mayaphi, Nokuzola Mbhele, Mpaphi B. Mbulawa, Adriano Mendes, Koleka Mlisana, Anele Mnguni, Thabo Mohale, Monika Moir, Kgomotso Moruisi, Mosepele Mosepele, Gerald Motsatsi, Modisa S. Motswaledi, Thongbotho Mphoyakgosi, Nokukhanya Msomi, Peter N. Mwangi, Yeshnee Naidoo, Noxolo Ntuli, Martin Nyaga, Lucier Olubayo, Sureshnee Pillay, Botshelo Radibe, Yajna Ramphal, Upasana Ramphal, James E. San, Lesley Scott, Roger Shapiro, Lavanya Singh, Pamela Smith-Lawrence, Wendy Stevens, Amy Strydom, Kathleen Subramoney, Naume Tebeila, Derek Tshiabuila, Joseph Tsui, Stephanie van Wyk, Steven Weaver, Constantinos K. Wibmer, Eduan Wilkinson, Nicole Wolter, Alexander E. Zarebski, Boitumelo Zuze, Dominique Goedhals, Wolfgang Preiser, Florette Treurnicht, Marietje Venter, Carolyn Williamson, Oliver G. Pybus, Jinal Bhiman, Allison Glass, Darren P. Martin, Andrew Rambaut, Simani Gaseitsiwe, Anne von Gottberg & Tulio de Oliveira. Rapid epidemic expansion of the SARS-CoV-2 Omicron variant in southern Africa. 2022. Nature. 10.1038/s41586-022-04411-y 

* Yuri Munsamy, Riaz Y. Seedat, Tumelo R. Sekee, **Phillip A. Bester**, Danelle van Jaarsveldt, Felicity J. Burt. Whole-Genome Sequence and Comparative Analysis of Human Papillomavirus Type 18 Isolated from a Nasopharyngeal Carcinoma from South Africa.  2021. Microbiology Resource Announcements.  https://doi.org/10.1128/MRA.00630-21 

* Yuri Munsamy, Riaz Y. Seedat, Tumelo R. Sekee, **Phillip A. Bester**, Felicity J. Burt.  Complete genome sequence of a HPV31 isolate from laryngeal squamous cell carcinoma and biological consequences for p97 promoter activity. 2021. PLOS ONE. https://doi.org/10.1371/journal.pone.0252524

- Oosthuizen Jaco, Kotze Maritha J, Van Der Merwe Nicole, Myburgh Ettienne J, **Bester Phillip**, van der Merwe Nerina C.  Rare BRCA2 Variants With Founder Haplotypes in the South African Population: Implications for Point-of-Care Testing Based on a Single-Institution BRCA1/2 Next-Generation Sequencing Study. 2021.  Frontiers in Oncology. 10(3403)  https://www.frontiersin.org/article/10.3389/fonc.2020.619469 

- Zhongning Zhao , Neil Heideman, **Phillip Bester**, Adriaan Jordaan and Margaretha D. Hofmeyr.  Climatic and topographic changes since the Miocene influenced the diversification and biogeography of the tent tortoise (Psammobates tentorius) species complex in Southern Africa.  2020.  BMC Evolutionary Biology.  https://doi.org/10.1186/s12862-020-01717-1
- Milton T. Mogotsi, Peter N. Mwangi, **Phillip A. Bester**, M. Jeffrey Mphahlele, Mapaseka L. Seheri, Hester G. O’Neill and Martin M. Nyaga.  Metagenomic Analysis of the Enteric RNA Virome of Infants from the Oukasie Clinic, North West Province, South Africa, Reveals Diverse Eukaryotic Viruses.  2020.  Viruses *12*(11), 1260.       https://doi.org/10.3390/v12111260
- S. Vawda, **P.A. Bester**, D. Morobadi, P. Matthews, J. Mokaya, D. Goedhals. Characteristics of individuals with hepatitis B virus infection in the Free State and Northern Cape provinces of South Africa.  2020. International Journal of Infectious Diseases.  DOI:https://doi.org/10.1016/j.ijid.2020.09.1363
- Leholonolo Mathengtheng, Dominique Goedhals, **Phillip A. Bester**, Jacqueline Goedhals, Felicity J. Burt.  Persistence of Crimean- Congo Hemorrhagic Fever  Virus RNA. 2020.  Emerging Infectious Diseases, 26(2), 385-387. https://dx.doi.org/10.3201/eid2602.191460.
- Micah Dimaculangan, Siewert C. Wiid, Phillip A. Bester, Tumelo R. Sekee, Felicity J. Burt.  A simple and rapid approach to prepare Sindbis and West Nile viral RNA controls for differentiation between positive samples and laboratory contamination. 2020. Journal of Virological Methods (278).  https://doi.org/10.1016/j.jviromet.2020.113822.
- Zhao, Z, Heideman, N, Grobler, P, Jordaan, **A, Bester**, P, Hofmeyr, MD. Unraveling the diversification and systematic puzzle of the highly polymorphic *Psammobates tentorius* (Bell, 1828) complex (Reptilia: Testudinidae) through phylogenetic analyses and species delimitation approaches. *J Zool Syst Evol Res*. 2020; 58: 308– 326. https://doi.org/10.1111/jzs.12338
- McNaughton AL,  Lourenço J, **Bester PA**, Mokaya J, Lumley SF, Obolski U, Forde D, Maponga TG, Katumba KR, Goedhals D, Gupta S, Seeley J, Newton R, Ocama P, 
  Matthews PC. Hepatitis B virus seroepidemiology data for Africa: Modelling intervention strategies based on a systematic review and meta-analysis. PLoS Med. 2020 Apr 21;17(4):e1003068. doi: 
  10.1371/journal.pmed.1003068. PMID: 32315297; PMCID: PMC7173646. 
- **Phillip Armand Bester**,  Andrie De Vries, Stephanus Riekert, Kim Steegen,  Gert van Zyl, Dominique Goedhals.  PhyloPi: An affordable, purpose built phylogenetic pipeline for the HIV drug resistance testing facility. 2019.  PloS ONE.  https://doi.org/10.1371/journal.pone.0213241
- Sabeehah Vawda, Dominique Goedhals, **Phillip Armand Bester**, Felicity Burt.  Seroepidemiologic Survey of Crimean-Congo Hemorrhagic Fever Virus in Selected Risk Groups, South Africa. July 2018. Emerging Infectious Diseases 24(7):1360-1363.   10.3201/eid2407.172096
- Riaz Seedat, C. E. Combrink, **Phillip Armand Bester**, Felicity Burt.  Determination of the complete genome and functional analysis of HPV6 isolate VBD19/10 from a patient with aggressive recurrent respiratory papillomatosis.      March 201. Epidemiology and Infection -1(10):1-8.  DOI: 10.1017/S0950268816000388
- Gregson J, Tang M, Ndembi N, Hamers RL, Rhee SY, Marconi VC, Diero L, Brooks K, Theys, K, Rinke de Wit TF, Arruda M, Garcia F, Monge S, Günthard, HF, Hoffmann, CJ, Kanki PJ, Kumarasamy, N, Kerschberger B, Mor O, Charpentier C, Todesco E, Rokx C, Gras L, Halvas EK, Sunpath H, Di Carlo D, Antinori A, Andreoni M, Latini A, Mussini C, Aghokeng A, Sonnerborg A, Ujjwal Neogi U, Fessel WJ, Agolory S, Yang C, Blanco JL, Juma JM, Smit, E, Schmidt, D, Watera C, Asio J, Kirungi W, Tostevin A, El– Hay T, Clumeck N, Goedhals D; van Vuuren C, **Bester PA**, Sabin C, Mukui I, Santoro MM, Perno CF, Hunt G, Morris L, Camacho R, de Oliveira T, Pillay D, Schulter E, Akio Murakami– Ogasawara A, Reyes– Terán G, Romero K, Avila– Rios S, Sirivichayakul, S, Ruxrungtham K, Mekprasan S, Dunn, D, Kaleebu, P, Raizes E, Kantor R, Shafer RW, Gupta RK on behalf of the TenoRes study group. (2016). [Global epidemiology of drug resistance following failure of WHO recommended first line regimens for adult HIV–1 infection – a multi–centre retrospective cohort study.](https://iris.ucl.ac.uk/iris/publication/1125368/1) Lancet Infect Dis 2016. May;16(5):565– 75.
- Goedhals D, **Bester PA**, Paweska JT, Swanepoel R, Burt FJ.  Comparative analysis of the L, M, and S RNA segments of Crimean-Congo haemorrhagic fever virus isolates from southern Africa. Journal of medical virology. 2015 Feb Vol 85, Issue 5 p717-724. DOI: 10.1002/jmv.24079
- Goedhals D, **Bester PA**, Paweska JT, Swanepoel R, Burt FJ. Next-generation sequencing of southern African Crimean-Congo haemorrhagic fever virus isolates reveals a high frequency of M segment reassortment. Epidemiology and Infection. 2014 May 1:1-11. [Epub ahead of print] PubMed PMID: 24786748.
- Manasa J, Lessells R, Rossouw T, Naidu K, Van Vuuren C, Goedhals D, van Zyl G, **Bester A**, Skingsley A, Stott K, Danaviah S, Chetty T, Singh L, Moodley P, Iwuji C, McGrath N, Seebregts CJ,  de Oliveira T. Southern African Treatment Resistance Network (SATuRN) RegaDB HIV drug resistance and clinical management database: supporting patient management, surveillance and research in southern Africa. Database (Oxford). 2014 Feb 6; 2014:bat082. doi: 10.1093/database/bat082. Print 2014. PubMed PMID: 24504151
- Borgonie, G., García-Moyano, A., Litthauer, D., Bert, W., **Bester, P.A.**, van Heerden, E and  Onstott, T.C.  (2011). Worms from hell: Nematoda from the terrestrial deep subsurface of South Africa.  Nature 2009-12-15946E, volume 474, issue 7349 from 02 June 2011.
- **Bester, P.A**., Litthauer, D., Piater, L.A., and van Heerden, E. (2010). A thioredoxin reductase-like protein from the thermophile, Thermus scotoductus SA-01, displaying iron reductase activity. FEMS Microbiology Letters 302, 182 -188.

### References

#### Prof. D Goedhals

Current employment at NHLS/UFS, Virology HOD
Affiliations:  NHLS Virology Pathologist
	                 Associate professor at the University of the Free State
Contact details:  gnvrdg@ufs.ac.za

#### Dr. C van Vuuren

Affiliations:  Internal Medicine HOD, 3 Military Hospital Bloemfontein
Contact details:  vanvuurenc@ufs.ac.za
